ci/test.sh

```diff
@@ -135,6 +135,21 @@ EOF
                 popd
 
                 rm -rf "${td}"
+                td=$(mktemp -d)
+                git clone \
+                    --depth 1 \
+                    --recursive \
+                    https://github.com/cross-rs/test-workspace "${td}"
+                
+                pushd "${td}"
+      cross run --target "${TARGET}" -p binary --manifest-path="./workspace/Cargo.toml"
+                pushd "workspace"
+                cross run --target "${TARGET}" -p binary
+                pushd "binary"
+                cross run --target "${TARGET}"
+                popd
+                popd
+                popd
```

Comment:
im not a bash expert, can you tell?

Is it possible to do the pops in one command?

Reply:
I think this is the best approach.

You can use `pushd -0 && dirs -c` to go to the bottom of the stack and then clear the stack, but this won't work if there's already a so it's better to just use `popd` multiple times IMO. I believe there's no correct way to do this using arguments to popd, since they just remove items from specific indexes in the `dirs` stack. This could be changed to a single line with `popd; popd; popd`.

The issue with it is as follows:

```bash
#!/bin/bash

call() {
	pushd .local
	pushd share
	pushd applications
	echo "$(ls -la .)"
	echo "dirs before clear: $(dirs)"
	pushd -0 && dirs -c
	# Correct way: popd; popd; popd
	echo "pwd inside call: $(pwd)"
}

pushd ~
call
echo "dirs after clear: $(dirs)"
echo "pwd outside call: $(pwd)"
```

This means that after `call`, we'll be back in the original directory, not in `~`, which is probably what was wanted.